### ISSUE TITLE
Changing volume from menus not working?

### ISSUE THREAD

**Issue by juanedi** [TheStalwart/Theremin#11]
Changing the volume from the volume slider works, but doing it from the menus (or keyboard shortcuts) doesn't.

This is the output config con my mpd.conf, I'm not sure it might be relevant. The server is running on a Debian machine.

audio_output {
    type        "alsa"
    name        "My ALSA Device"
    device      "hw:0,0"    # optional
    format      "44100:16:2"    # optional
    mixer_device    "default"   # optional
    mixer_control   "PCM"       # optional
    mixer_index "0"     # optional
}

Please excuse me if this isn't an issue with Theremin.

**Comment by TheStalwart** [comment]
Yep, they aren't. I believe, i tried to debug an issue, but problem wasn't obvious, and since i usually use two-finger gesture to change volume, i'm not affected by the bug. Sorry for that :)
